source/common/http/conn_manager_utility.cc

```diff
@@ -223,12 +229,17 @@ void ConnectionManagerUtility::mutateXfccRequestHeader(Http::HeaderMap& request_
         // The "SAN" key still exists even if the SAN is empty.
         client_cert_details.push_back("SAN=" + connection.ssl()->uriSanPeerCertificate());
```

Comment:
Do you want to keep this for backward compatibility? It's the same as the "Http::ClientCertDetailsType::URI" case.

Reply:
There was a discussion during SAN deprecation proposal and the conclusion was to keep it backwards compatible for now. Envoy follows breaking changes, so I guess it should be okay to deprecate SAN completely in a follow-up PR? Maybe give it sometime and then deprecate it before releasing 1.7.0?

Reply:
SGTM. Thanks.